digdag-cli/src/main/java/io/digdag/cli/client/ShowWorkflow.java

```diff
@@ -51,53 +54,42 @@ private void showWorkflows(String projName)
     {
         DigdagClient client = buildClient();
 
+        CollectionPrinter<RestWorkflowDefinition> formatter = new CollectionPrinter<>();
+        formatter.column("PROJECT", wf -> wf.getProject().getName());
+        formatter.column("PROJECT ID", wf -> Integer.toString(wf.getProject().getId()));
```

Comment:
I think `PROJECT_ID` (underscore) is better because space looks like a different column.